app/jobs/dqt/sync_states_job.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Dqt
+  class SyncStatesJob < ApplicationJob
+    queue_as :default
+    retry_on Client::HttpError
+
+    BATCH_SIZE = 100
```

Comment:
Is it worth making these two configurable or optional arguments that you can pass in or configurable optional arguments that you can pass in? 😆